### PR TITLE
chore: reorganize frontend file structure

### PR DESCRIPTION
**Pull request by joyce-shi**
## Notion ticket link

[Clean Frontend File Structure](https://www.notion.so/uwblueprintexecs/Clean-Frontend-File-Structure-e333aeb27be741e3a07705aa67530007)

## Implementation description
* Added folders

## Steps to test
1. Please pull the branch to make sure that there are no breaking changes!

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,3 +2,7 @@
 # yarn lockfile v1
 
 
+"@types/react-select-country-list@^2.2.1":
```
Comment (carissa-tang) [inline yarn.lock]: should this be added to `frontend/yarn.lock` ? it should also update `package.json` after you run yarn add

Comment (joyce-shi) [reply yarn.lock]: i think you're right! updating now

File: frontend/tsconfig.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "noImplicitAny": false
```
Comment (carissa-tang) [inline frontend/tsconfig.json]: is there a particular reason you added this? i feel like its probably better to have stricter type checking?

Comment (joyce-shi) [reply frontend/tsconfig.json]: LOL i don't remember why so i removed it (:

**Review by carissa-tang — APPROVED** [review]

**Pull request merged**